ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutor.java

```diff
@@ -365,9 +365,9 @@ private void processRecordsForTopic(
       final TopologyTestDriver topologyTestDriver,
       final Topic sinkTopic
   ) {
-    final int idx = 0;
+    int idx = 0;
```

Comment:
Look mar, a bug that would have been found it `final` was required when the code was first submitted! :p